docs/changelogs/v0.30.md

```diff
@@ -129,6 +130,10 @@ Modification time support was also added to the Gateway. If present, value from
 > [!NOTE]
 > Storing `mode` and `mtime` requires root block to be `dag-pb` and disabled `raw-leaves` setting to create envelope for storing the metadata.
 
+#### Commands Preserve Specified Hostname
+
+When executing a comman, if a hostname is specified by `--api=/dns4/<domain>/` the resulting HTTP request now contains the hostname, instead of the the IP address that the hostname resolved to, as was the previous behavior. This makes it easier for those trying to run kubo behind a reverse proxy using hostname-based rules.
```

Comment:
```suggestion
When executing a [CLI command](https://docs.ipfs.tech/reference/kubo/cli/) over [Kubo RPC API](https://docs.ipfs.tech/reference/kubo/rpc/), if a hostname is specified by `--api=/dns4/<domain>/` the resulting HTTP request now contains the hostname, instead of the the IP address that the hostname resolved to, as was the previous behavior. This makes it easier for those trying to run Kubo behind a reverse proxy using hostname-based rules.
```